src/main/java/org/codehaus/plexus/components/io/resources/PlexusIoURLResource.java

```diff
@@ -55,5 +56,4 @@ public String getDescriptionForError( URL url )
         return url != null ? url.toExternalForm() : "url=null";
     }
 
-    public abstract URL getURL() throws IOException;
```

Comment:
Because is declared in the interface PlexusIoResource:
https://github.com/codehaus-plexus/plexus-io/blob/306612ce651f15f5c13612555a91b7b592626996/src/main/java/org/codehaus/plexus/components/io/resources/PlexusIoResource.java#L92

So the hierarchy of this is PlexusIoURLResource extends AbstractPlexusIoResource implements PlexusIoResource, and since is declared in the interface, the method is redundant in the PlexusIoURLResource abstract class.